pkg/gather/clusterconfig/clusterconfig.go

```diff
@@ -665,6 +669,35 @@ func GatherCertificateSigningRequests(i *Gatherer) func() ([]record.Record, []er
 	}
 }
 
+// GatherCRD collects the specified Custom Resource Definitions.
+//
+// The following CRDs are gathered:
+// - volumesnapshots.snapshot.storage.k8s.io (10745 bytes)
+// - volumesnapshotcontents.snapshot.storage.k8s.io (13149 bytes)
+//
+// The CRD sizes above are in the raw (uncompressed) state.
+//
+// Location in archive: config/crd/
+func GatherCRD(i *Gatherer) func() ([]record.Record, []error) {
+	return func() ([]record.Record, []error) {
+		crds, err := i.crdClient.CustomResourceDefinitions().List(metav1.ListOptions{})
```

Comment:
Would it be possible to instead of getting whole List, just get concrete CRD by Name ? It could be a bit more efficient I think.
Maybe something like:
```
crds := []string{"one","two"}
  for _, c := range crds {
    i.crdClient.CustomResourceDefinitions().Get(c,metav1.ListOptions{})
//... append
  }
```

Reply:
Oh, thanks for pointing this out, I totally didn't realize we could do this.